BUILDING.md

```diff
@@ -75,22 +86,26 @@ Node.js does not support a platform version if a vendor has expired support
-| macOS/OS X   | Tier 1       | >= 10.11                        | x64              |                               |
-| Windows      | Tier 1       | >= Windows 7/2008 R2/2012 R2    | x86, x64         | [1](#fn1),[2](#fn2),[3](#fn3) |
-| SmartOS      | Tier 2       | >= 16                           | x64              |                               |
-| FreeBSD      | Tier 2       | >= 11                           | x64              |                               |
-| GNU/Linux    | Tier 2       | kernel >= 3.13.0, glibc >= 2.19 | ppc64le >=power8 |                               |
-| AIX          | Tier 2       | >= 7.1 TL04                     | ppc64be >=power7 |                               |
-| GNU/Linux    | Tier 2       | kernel >= 3.10, glibc >= 2.17   | s390x            |                               |
-| GNU/Linux    | Experimental | kernel >= 2.6.32, glibc >= 2.12 | x86              | limited CI                    |
-| Linux (musl) | Experimental | musl >= 1.0                     | x64              |                               |
-
-<em id="fn1">1</em>: Tier 1 support for building on Windows is only on 64-bit
-  hosts. Support is experimental for 32-bit hosts.
+| Operating System | Architectures    | Versions                        | Support Type | Notes                             |
+| ---------------- | ---------------- | ------------------------------- | ------------ | --------------------------------- |
+| GNU/Linux        | x64              | kernel >= 3.10, glibc >= 2.17   | Tier 1       | e.g. Ubuntu 16.04, Debian 9, EL 7 |
+| GNU/Linux        | x64              | kernel >= 3.10, musl >= 1.1.19  | Tier 2       | e.g. Alpine 3.8                   |
+| GNU/Linux        | x86              | kernel >= 3.10, glibc >= 2.17   | Experimental |                                   |
+| GNU/Linux        | arm64            | kernel >= 4.5, glibc >= 2.17    | Tier 1       | e.g. Ubuntu 16.04, Debian 9, EL 7, [1](#fn1) |
+| GNU/Linux        | armv7            | kernel >= 4.14, glibc >= 2.24   | Tier 1       | e.g. Ubuntu 18.04, Debian 9       |
+| GNU/Linux        | armv6            | kernel >= 4.14, glibc >= 2.24   | Experimental | Downgraded as of Node.js 12       |
+| GNU/Linux        | ppc64le >=power8 | kernel >= 3.13.0, glibc >= 2.19 | Tier 2       | e.g. Ubuntu 16.04, EL 7           |
+| GNU/Linux        | s390x            | kernel >= 3.13.0, glibc >= 2.19 | Tier 2       | e.g. EL 7                         |
+| Windows          | x64, x86 (WoW64) | >= Windows 7/2008 R2/2012 R2    | Tier 1       | [2](#fn2),[3](#fn3)               |
+| Windows          | x86 (native)     | >= Windows 7/2008 R2/2012 R2    | Experimental |                                   |
+| Windows          | arm64            | >= Windows 10                   | Experimental |                                   |
+| macOS            | x64              | >= 10.11                        | Tier 1       |                                   |
```

Comment:
TODO: pending discussion

Reply:
We one one indicator that V8 7.4 doesn't compile With XCode 8, only with 9:

XCode 8 (10.11) - https://ci.nodejs.org/job/node-test-commit-osx/25502/nodes=osx1011/ does compile
XCode 9 (10.12) - https://ci.nodejs.org/job/temp-node-test-commit-osx/1/nodes=osx1012/ fails just 1 test

Trying again with 8/10.11 - https://ci.nodejs.org/job/node-test-commit-osx/25637/nodes=osx1011/

Reply:
+1 to bump to 10.12. It's already more than two years old and I think 10.11 is not supported by Apple anymore

Reply:
10.12 it is